pkgs/development/python-modules/httplib2/default.nix

```diff
@@ -40,7 +41,7 @@ buildPythonPackage rec {
   ];
 
   # Don't run tests for Python 2.7
-  doCheck = !isPy27;
+  doCheck = !(isPy27 || stdenv.isDarwin);
```

Comment:
do you mind adding a comment as to why darwin is skipped for tests?

Reply:
Also please modify the comment above to contain a useful information.

Reply:
Thanks. I improved the comment.

Reply:
Did you try ``__darwinAllowLocalNetworking = true;``?

Reply:
I just tried it, and it did not work: 
```
tests/test_auth.py:98: in test_digest_next_nonce_nc
    with tests.server_request(handler, request_count=5) as uri:
        grenew_nonce = [<function http_reflect_with_auth.<locals>.renew_nonce at 0x112095700>]
        handler    = <function http_reflect_with_auth.<locals>.http_reflect_with_auth_handler at 0x1120955e0>
        http       = <httplib2.Http object at 0x112000be0>
        password   = "l%N +\x15[a3yFCk'f}eW}H+y&Z\x02]\x7fx8f\x14\x14%zA~CI$(an\x04Cz71\x178VDh;\x10R\x18QK\x185SH]"
/nix/store/xf3244kbn36z04yw98mpgvvsgdcbz438-python3-3.8.8/lib/python3.8/contextlib.py:113: in __enter__
    return next(self.gen)
        self       = <contextlib._GeneratorContextManager object at 0x11208b760>
tests/__init__.py:325: in server_socket
    server.bind((bind_hostname, 0))
E   socket.gaierror: [Errno 8] nodename nor servname provided, or not known
```